datastore/tests/unit/value_test.py

```diff
@@ -0,0 +1,188 @@
+from datetime import datetime
+
+import pytest
+
+from gcloud.rest.datastore import Key
+from gcloud.rest.datastore import PathElement
+from gcloud.rest.datastore import Value
+
+
+class TestValue(object):
+    @staticmethod
+    @pytest.mark.parametrize('json_key,json_value', [
+        # TODO Make this Python 2 compatible
```

Comment:
I think it's non-trivial and I wanted to focus on other things for this PR. I'll add a ticket for myself if we decide to merge the TODO